reimbursement/templates/include/reimbursement.html

```diff
@@ -3,17 +3,46 @@
 
 {% if application.scholarship %}
     <h3>Travel reimbursement</h3>
-    {% if reimbursement and reimbursement.is_sent %}
+    {% if not reimbursement %}
+        Unfortunately, we don't have enough budget to cover your travel expenses. We would like to have enough money for
+        everyone but our budget is limited. Hope you can still make it! If you can't make it, please cancel your
+        application invite.
+    {% elif reimbursement.is_expired %}
+        Unfortunately your travel reimbursement has expired. We sent your travel reimbursement
+        {{ reimbursement.status_update_date|timesince }} ago but you haven't uploaded a valid ticket. Send us an email to
+        {{ static_values.email|urlize }} if you
+        think this is an error.
+        {% if reimbursement.change_reason %}
+            <p><b>Reason</b>: {{ reimbursement.change_reason }}</p>
+        {% endif %}
+    {% elif reimbursement.is_sent %}
         <p>You have been granted travel reimbursement for your trip from {{ reimbursement.origin_city }}
             ({{ reimbursement.origin_country }}).</p>
 
         <h4>Amount granted: {{ reimbursement.assigned_money }}{{ reimbursement.currency_sign }}</h4>
-        <p>Make sure to submit your ticket before it expires</p>
-        <a href="{{ reimbursement.get_form_url }}" target="_blank" class="btn btn-success btn-block">Submit your ticket</a>
+        <p>You have {{ reimbursement.expiration_time|timeuntil }} to upload your ticket, or your travel reimbursement
+            will expire.</p>
+        <a href="{{ reimbursement.get_form_url }}" target="_blank" class="btn btn-success btn-block">Submit your
+            ticket</a>
+        <p>The review process is done manually, so it may take a while to be processed.  We apologize for the inconvenience.</p>
+    {% elif reimbursement.friend_accepted %}
+        <p>A friend uploaded a receipt that covered your travel. Let us know if you think this is an error.</p>
+    {% elif reimbursement.is_accepted %}
+        <p>Your travel reimbursements tickets have been accepted. Here is the amount that we will reimburse to you (and any friend your ticket included):</p>
```

Comment:
Should be `your travel **reimbursement** tickets`, plural sounds weird.
Also I like better `and any friend included in your ticket` but that's just personal :)